lhotse/supervision.py

```diff
@@ -512,6 +513,89 @@ def from_dicts(data: Iterable[Dict]) -> "SupervisionSet":
             SupervisionSegment.from_dict(s) for s in data
         )
 
+    @staticmethod
+    def from_rttm(path: Pathlike) -> "SupervisionSet":
+        """
+        Read an RTTM file located at ``path`` and create a :class:`.SupervisionSet` manifest for them.
+        Can be used to create supervisions from custom RTTM files (see, for example, :class:`lhotse.dataset.DiarizationDataset`).
+
+        .. hint::
+
+            Use the :meth:`.from_rttms` method if you want to read multiple RTTM files at once.
+
+        The following description is taken from the [dscore](https://github.com/nryant/dscore#rttm) toolkit:
+
+        Rich Transcription Time Marked (RTTM) files are space-delimited text files
+        containing one turn per line, each line containing ten fields:
+
+        - ``Type``  --  segment type; should always by ``SPEAKER``
+        - ``File ID``  --  file name; basename of the recording minus extension (e.g.,
+        ``rec1_a``)
+        - ``Channel ID``  --  channel (1-indexed) that turn is on; should always be
+        ``1``
+        - ``Turn Onset``  --  onset of turn in seconds from beginning of recording
+        - ``Turn Duration``  -- duration of turn in seconds
+        - ``Orthography Field`` --  should always by ``<NA>``
+        - ``Speaker Type``  --  should always be ``<NA>``
+        - ``Speaker Name``  --  name of speaker of turn; should be unique within scope
+        of each file
+        - ``Confidence Score``  --  system confidence (probability) that information
+        is correct; should always be ``<NA>``
+        - ``Signal Lookahead Time``  --  should always be ``<NA>``
+
+        For instance:
+
+            SPEAKER CMU_20020319-1400_d01_NONE 1 130.430000 2.350 <NA> <NA> juliet <NA> <NA>
+            SPEAKER CMU_20020319-1400_d01_NONE 1 157.610000 3.060 <NA> <NA> tbc <NA> <NA>
+            SPEAKER CMU_20020319-1400_d01_NONE 1 130.490000 0.450 <NA> <NA> chek <NA> <NA>
+
+        :param path: Path to RTTM file.
+        :return: a new ``SupervisionSet`` instance containing segments from the RTTM file.
+        """
+        segments = []
+        with open(path, "r") as f:
+            for idx, line in enumerate(f):
+                parts = line.strip().split()
+                assert len(parts) == 10, f"Invalid RTTM line: {line}"
+                recording_id = parts[1]
+                segments.append(
+                    SupervisionSegment(
+                        id=f"{recording_id}-{idx:06d}",
+                        recording_id=recording_id,
+                        channel=int(parts[2]),
+                        start=float(parts[3]),
+                        duration=float(parts[4]),
+                        speaker=parts[7],
+                    )
+                )
+        return SupervisionSet.from_segments(segments)
+
+    @staticmethod
+    def from_rttms(paths: Union[Pathlike, Iterable[Pathlike]]) -> "SupervisionSet":
```

Comment:
I meant if we can merge `from_rttms` and `from_rttm` into a single method that auto-detects the right thing to do with a single path vs an iterable of them. The names differ by just one character which makes them easy to confuse. WDYT?

Reply:
Yeah that makes sense. I'll make that change.